### PR TITLE
0.6.0 updates v3

### PR DESCRIPTION
**Pull request by finglis**
Ok to merge, some pages hidden that are 0.6.0 only. Will resume edits post 12k.

### CODE REVIEW

File: docs/intro/formats.md [hunk 1]
```diff
@@ -165,6 +173,8 @@ It is already possible to add URLs in a variety of formats that OMERO can provid
 This makes it possible to import multiple images -- even including some annotations -- to a QuPath {doc}`project <../tutorials/projects>` in one go.
 :::
 
+***TODO*** Confirm these details are all still relevant with Leos new Omero extension
+
```
Comment (lauranicolass) [inline docs/intro/formats.md:177]: Also confirm that the QuPath issues page is still relevant? Last entry is from 2020

File: docs/intro/acknowledgements.md [hunk 2]
```diff
@@ -109,6 +109,8 @@ The sources and terms of reuse for these images are summarized in the table belo
   - CC-BY 4.0
 ::::
 
+***TODO*** Add TMA image to the table
```
Comment (lauranicolass) [inline docs/intro/acknowledgements.md:112]: Add a line explaining that to access the images one has to follow the links, not download the miniature? There was some trouble with this at the Crick

File: docs/intro/formats.md [hunk 3]
```diff
@@ -165,6 +173,8 @@ It is already possible to add URLs in a variety of formats that OMERO can provid
...
 ### Community extensions
```
Comment (lauranicolass) [inline docs/intro/formats.md:178]: For Updating Extensions, I would add the Add: Extensions ‣ Manage extensions ‣ Open extension directory instruction. I know its above, but if someone only clicks on Updating extansion, they won’t see it

File: docs/intro/acknowledgements.md (hunk 2 above)
Comment (finglis) [reply docs/intro/acknowledgements.md:112]: Noted for post I2K

File: docs/intro/formats.md [hunk 4]
```diff
@@ -133,13 +133,21 @@ Consequently, although OpenSlide and Bio-Formats support many TIFF files, it is
 Perhaps the most common reason for this is that the file does not contain pyramidal layers, or these layers cannot be automatically recognized.
 This is one reason why well-supported, open formats should generally be preferred (e.g. OME-TIFF).
 
+#### OME-Zarr
+QuPath version 0.6.0 introduced support for reading and writing [OME-Zarr](https://link.springer.com/article/10.1007/s00418-023-02209-1) images. This file type was developed by the OME team in collaboration with many individuals and institutes to address the need for a scalable and cloud-friendly large image format.
+
+***TODO*** Add more details about OME-Zarr by some that knows more about it. May need to move or add in additional section within the OMERO section...?
+
 ## Open URI
```
[the remark this answers is not in the document]
Comment (finglis) [reply docs/intro/formats.md:141]: Noted for post I2K

File: docs/intro/formats.md (hunk 1 above)
Comment (finglis) [reply docs/intro/formats.md:177]: Noted for post I2K

File: docs/intro/formats.md (hunk 3 above)
Comment (finglis) [reply docs/intro/formats.md:178]: Noted for post I2K

**Comment by finglis** [comment]
Now in a merge-able state however screenshots are taken in 0.6.0-rc2 so can wait till its full release if preferred. 
Instanseg, detailed release notes and tips and tricks pages hidden for now till each have their issues addressed (see loop page).

**Pull request merged**